emoji_list.py

```diff
@@ -0,0 +1,2038 @@
+# flake8: noqa
+# Source originally taken from https://github.com/sevenc-nanashi/discord-emoji.
+# Modified to remove the UNICODE_TO_DISCORD dictionary.
+
+DISCORD_TO_UNICODE = {
```

Comment:
@Cephian we may want to curate this list a bit.

Reply:
And by a bit I mean a lot. Some of them look broken:

![image](https://user-images.githubusercontent.com/1342360/148866286-8b022ade-061a-42fa-a009-3ae515d4e501.png)

And others don't really do anything other than be a bit distracting:

![image](https://user-images.githubusercontent.com/1342360/148866378-b7b1d65d-b662-4027-af9b-1d32ffb129d3.png)

Perhaps we should instead start with a whitelist of appropriate/fun(ny) emoji?

Reply:
Yes, a whitelist is a good idea. The reason why some emoji look broken is because it seems Discord uses your system font to render emoji in nicknames and not the Discord emoji images.

I notice since this is a Discord emoji list it follows the sorting within Discord, so along with that I propose that for now we keep all emoji in the first three sections (People, Nature, Food) and drop other ones. This would just remove all emoji from "soccer" onward from the list.

We can always hand-pick a better list later of course, or even now if you feel up to it (I don't particularly atm).